pre_commit/commands/autoupdate.py

```diff
@@ -128,7 +132,7 @@ def autoupdate(runner, store, tags_only, repos=()):
         try:
             new_repo_config = _update_repo(repo_config, store, tags_only)
         except RepositoryCannotBeUpdatedError as error:
-            output.write_line(error.args[0])
+            output.write_line(str(error))
```

Comment:
this should still be `e.args[0]` right?  calling `str()` can lead to encoding problems in python2.x

Reply:
Yep, but `e.args[0]` was giving me issue with `write_line`, namely `five.to_bytes(s)`. Any suggestions?

Reply:
After moving the error handling logic up, this is not an issue any more. Strange.